### PR TITLE
A multitude of improvements!

### PR DESCRIPTION
**Pull request by Gitoffthelawn**
As a gesture of appreciation, I spent some time greatly improving this document.  Thank you for Dark Reader and all your hard work on it!

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,62 +1,69 @@
 <h2 align="center">Contributing</h2>
 
-<p align="center">Contributing to Dark Reader can be in a variety of ways. Here, you can learn more about it. Thank you in advance for reading this <code>CONTRIBUTING.md</code>.</p>
+<p align="center">YOU can contribute to Dark Reader in a variety of ways!  Here, you can learn more.  Thank you in advance for reading this document.</p>
```
Comment (Gusted) [inline CONTRIBUTING.md:3]: I always felt so weird with reading documents that tries to get your attraction by capitalizing certain words and using `!` - Feels like a TV ad that was on a budget.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,62 +1,69 @@
...
-[Improve or suggest](https://github.com/darkreader/darkreader/tree/master/src/_locales) a translation.
+[Improve or suggest](https://github.com/darkreader/darkreader/tree/master/src/_locales) a translation.  
 See the list of [language codes](https://developer.chrome.com/webstore/i18n#localeTable) that we can support.
 
-## Fix a wrong inversion!
+## Fix a Wrong Inversion!
 
-If a website is **already dark**, that regardless of system's preferred color-scheme has all pages dark by default. You can **add it to the [dark-sites.config](https://github.com/darkreader/darkreader/blob/master/src/config/dark-sites.config) file**
-*(please, preserve the alphabetical order)*.
+If a website is **already dark** (meaning that regardless of the system's preferred color-scheme, has all pages dark by default), you can **add it to the [dark-sites.config](https://github.com/darkreader/darkreader/blob/master/src/config/dark-sites.config) file** *(please, preserve alphabetical order)*.
```
Comment (Gusted) [inline CONTRIBUTING.md:18]: "already dark" shouldn't mean/equal that regardless [...].

Let's try something along the lines of `If a website is **by-default** darkened ([...])`
That makes more sense to me.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -103,19 +111,20 @@ CSS
...
 
 Here is a full table of available CSS variables:
 
 | Variable | Description | Use |
 |-|-|-|
-| `--darkreader-neutral-background` | Neutral background color that corresponds to the user's settings. | Mostly used for elements that have a wrong background color |
-| `--darkreader-neutral-text` | Neutral text color that corresponds to the user's settings. | Used for elements with a wrong text color |
-| `--darkreader-selection-background` | The text color setting defined by the user. | The user's Text Color setting |
-| `--darkreader-selection-text` | The background color setting defined by the user. | The user's Background Color setting |
+| `--darkreader-neutral-background` | Neutral background color corresponding to the user's settings | Mostly used for elements with a wrong background color |
```
Comment (Gusted) [inline CONTRIBUTING.md:120]: Not sure if changing `that corresponds` -> `corresponding` is correct grammar - More like doesn't read the sentence smoothly.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -140,43 +149,45 @@ CSS
...
 - `REMOVE BG` removes the background image from an element and forces a black background.
 
-## Add new features or fix bugs!
+## Add New Features or Fix Bugs!
+
+If you would like to **add a new feature** to Dark Reader or **fix a bug**, please **submit an issue** in GitHub (if there is no existing one), **discuss** it with active contributors, and wait for **approval**.
 
-If you would like to **add a new feature** to Dark Reader or **fix a bug**, **submit an issue** in GitHub (if there is no existing one), **discuss** it with active contributors, and wait for **approval**.
+#### To build and debug the extension:  
+1. Install the [Node.js](https://nodejs.org/) LTS.
```
Comment (Gusted) [inline CONTRIBUTING.md:161]: `LTS`-> `LTS Version`

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -140,43 +149,45 @@ CSS
...
-If you would like to **add a new feature** to Dark Reader or **fix a bug**, **submit an issue** in GitHub (if there is no existing one), **discuss** it with active contributors, and wait for **approval**.
+#### To build and debug the extension:  
+1. Install the [Node.js](https://nodejs.org/) LTS.  
+1. Install development dependencies by running `npm install` in the project's root folder.  
+1. Execute `npm run debug`.
 
-To build and debug the extension **install the [Node.js](https://nodejs.org/)** LTS.  
-Install development dependencies by running `npm install` in the project root folder.  
-Then execute `npm run debug`.
+#### To install your test extension in Chrome and Edge
```
Comment (Gusted) [inline CONTRIBUTING.md:165]: Repeating `1.`

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -140,43 +149,45 @@ CSS
...
+1. Navigate to the project's `debug/` folder.
 
-#### Chrome and Edge
-- Open the `chrome://extensions` page.
-- Disable the official Dark Reader version.
-- Enable the **Developer mode**.
-- Click **Load unpacked extension** button
-- Navigate to the project's `debug/` folder.
+#### To install your test extension in Firefox
+1. Open the `about:addons` page.
```
Comment (Gusted) [inline CONTRIBUTING.md:174]: Repeating `1.`

Also this step plus the disable official one is unnecessary - Firefox handles this fine.

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -140,43 +149,45 @@ CSS
...
+1. Disable the official Dark Reader version.
+1. Open the `about:debugging#addons` page.
+1. Click **Load Temporary Add-on** button.
+1. Select the test extension you built.
```
Comment (Gusted) [inline CONTRIBUTING.md:178]: This step + next step sounds a bit conflicting to me.

**Review by Gusted — CHANGES_REQUESTED** [review]
Minor things - Great improvements, thank you!

**Comment by Gitoffthelawn** [comment]
@Gusted You're welcome and thanks for all the great feedback!  I'll go through each item and make edits when I have some more time.

File: CONTRIBUTING.md (hunk 1 above)
Comment (joshdaniel87) [reply CONTRIBUTING.md:3]: > I always felt so weird with reading documents that tries to get your attraction by capitalizing certain words and using `!` - Feels like a TV ad that was on a budget.

**Pull request closed without merging**